CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- edited README.md ([#184](https://github.com/giantswarm/external-dns-app/pull/184))
+
```

Comment:
Personally, I would omit adding a changelog entry in this case because its not a change in the actual chart that needs to be known in case someone upgrades to this version. I'm interested in other opinions.

Reply:
In case you want to keep the entry, please make sure it tells what the change in README.md was

Reply:
I basically agree with @ubergesundheit. I think a change log entry is not required for stuff like this but also not forbidden.

Reply:
I will remove the entry